ktor-client/ktor-client-tests/common/test/io/ktor/client/tests/plugins/CookiesIntegrationTests.kt

```diff
@@ -241,5 +241,23 @@ class CookiesIntegrationTests : ClientLoader() {
         }
     }
 
+    @Test
+    fun testInvalidCharactersInCookie() = clientTests(listOf("Js")) {
```

Comment:
We already have a test checking this behavior: `RenderSetCookieTest.renderCookieThrowsOnNotEncodedExtensions()[jvm]`
Could you change it instead of adding a new one?